core/src/main/java/com/xmartlabs/bigbang/core/helper/ui/IntentHelper.kt

```diff
@@ -86,9 +86,9 @@ object IntentHelper {
     intent.type = "message/rfc822"
     intent.putExtra(Intent.EXTRA_EMAIL, arrayOf(email))
     
-    subject?.takeIf(String::isNotEmpty).let { intent.putExtra(Intent.EXTRA_SUBJECT, it) }
-    body?.takeIf(String::isNotEmpty).let { intent.putExtra(Intent.EXTRA_TEXT, it) }
-    filePath?.takeIf(String::isNotEmpty).let { intent.putExtra(Intent.EXTRA_STREAM, Uri.parse("file://" + it)) }
+    subject?.takeIf(String::isNotEmpty)?.let { intent.putExtra(Intent.EXTRA_SUBJECT, it) }
+    body?.takeIf(String::isNotEmpty)?.let { intent.putExtra(Intent.EXTRA_TEXT, it) }
+    filePath?.takeIf(String::isNotEmpty)?.let { intent.putExtra(Intent.EXTRA_STREAM, Uri.parse("file://" + it)) }
```

Comment:
haven't you got problems with android 6 permissions?

Reply:
I don't think you need any permission for this per-se. But the file is private to the application this way (i think)